### ISSUE TITLE
Game freezes when loading large audio files

### ISSUE THREAD

**Issue by thenml** [undefined06855/EditorMusic#23]
The game experiences a noticeable freeze lasting 3-7 seconds when loading large audio files (e.g., files larger than 80mb). I put entire playlists in the config, with the folder containing 1.2 GB of audio across just 10 files.

While this is not that big of a deal, it does slow down playtesting since the audio files are reloaded each time the editor is opened.

My suggestion would be to implement loading audio on a separate thread (if possible, i dont know much about cpp / geode modding).

**Comment by undefined06855** [comment]
yeah I think loading them in a separate thread is a good idea, definitely possible if fmod allows, in the meantime just have smaller audio files 😭

**Comment by undefined06855** [comment]
mod is being rewritten, this will be fixed in the rewrite
